include/RAJA/exec-sequential/reduce_sequential.hxx

```diff
@@ -146,12 +144,8 @@ private:
   //
   ReduceMin<seq_reduce, T>();
 
-  bool m_is_copy;
-  int m_myID;
-
-  T m_reduced_val;
-
-  CPUReductionBlockDataType* m_blockdata;
+  const my_type * parent;
+  mutable T val;
```

Comment:
Can we continue to use the convention that class member variables have "m_" prefix?